doc/dev/binary-file-formats.rst

```diff
@@ -147,5 +147,3 @@ Table size          8         0   Length of table, **in bits** (``uint64_t``).
 Bins                N/8+1     8   This table's bytes, length given by previous
                                   field, divided by 8, plus 1 (``uint8_t``).
 ================== ======= ===== ==============================================
-
-.. todo:: Document ``Tags``, ``Stoptags``, ``Subset``, ``Labelset``
```

Comment:
It builds just fine when this line is removed. See https://readthedocs.org/projects/khmer/builds/5435660/.

This definitely seems like a bug/version problem on ReadTheDocs' end, but probably not worth investing any time troubleshooting. I propose we move this TODO to an issue (I assume it's still relevant) and keep this change. Any comments?